src/mxarray.jl

```diff
@@ -636,16 +636,16 @@ function jsparse(mx::MxArray)
 end
 
 
-function jstring(mx::MxArray)
+function String(mx::MxArray)
     if !(classid(mx) == mxCHAR_CLASS && ((ndims(mx) == 2 && nrows(mx) == 1) || is_empty(mx)))
-        throw(ArgumentError("jstring only applies to strings (i.e. char vectors)."))
+        throw(ArgumentError("only applies to strings (i.e. char vectors)"))
```

Comment:
This doesn't look like the right way to fix the error message. It's unclear what causes the error. Same below.